india_compliance/patches/post_install/check_version_compatibility.py

```diff
@@ -0,0 +1,32 @@
+from packaging import version
+
+import frappe
+
+# Note: India Compliance versions in DECENDING order only
+VERSION_COMPATIBILITY = {
+    # Development Versions
+    "15.0.0-dev": {"frappe": "15.0.0-dev", "erpnext": "15.0.0-dev"},
+    # Stable Versions
+    "15.0.0": {"frappe": "15.0.0", "erpnext": "15.0.0"},
+    "14.12.0": {"erpnext": "14.29.0"},
+    "14.10.4": {"erpnext": "14.29.0"},
+    "14.0.0": {"frappe": "14.0.0", "erpnext": "14.0.0"},
+}
+
+
+def execute():
+    ic_version = frappe.get_attr("india_compliance.__version__")
```

Comment:
`current_version`. And why are you using `frappe.get_attr`? that is so unnecessary when you can just import.